neural_speed/core/layers/bestla_common.hpp

```diff
@@ -26,12 +26,15 @@ namespace ne_bestla {
 class ne_threading {
  public:
   static bestla::parallel::IThreading* get() {
+    GetCPUDevice();
+    static bestla::parallel::StdThreading OptmizedThreading;
 #ifdef NS_USE_OMP
-    static bestla::parallel::OMPThreading DefaultThreading(4);
-#else
-    static bestla::parallel::StdThreading DefaultThreading(4);
+    static bestla::parallel::OMPThreading DefaultThreading;
+    if (!_cd->isHybrid()) {
+      return &DefaultThreading;
```

Comment:
emm, if user uses hybrid CPUs but they only launch P-Core, will this choose affect performance?

Reply:
This case is same as server. `StdThreading` has the same performance as `OMPThreading` in first token by out test on SPR.